backend/test/capsule.unit.test.ts

```diff
@@ -0,0 +1,71 @@
+import { BigNumber, Contract, ContractFactory } from "ethers";
+import { expect, assert } from "chai";
+const { ethers, network } = require("hardhat");
+
```

Comment:
//Here you need to import the specific contract types from the typechain
//example
import { GenericERC20, NftCapsule, Erc20Capsule, MixedCapsule } from "../build/typechain/"